### PR TITLE
CMR-5489: Fixed SES compression rate retrieval for Var1.4

### PR DESCRIPTION
**Pull request by sxu123**
This ticket only addresses the existing compression formats: ASCII and NetCDF-4.  Since we don't use compression rate for ASCII estimate anymore, it only affects NetCDF-4.

### CODE REVIEW

File: cmr-exchange/ous-plugin/project.clj [hunk 1]
```diff
@@ -15,28 +15,28 @@
...
+                 [clojusc/twig "0.4.1"]
+                 [com.stuartsierra/component "0.4.0"]
                  [environ "1.1.0"]
                  [gov.nasa.earthdata/cmr-authz "0.1.1"]
                  [gov.nasa.earthdata/cmr-exchange-common "0.2.2"]
                  [gov.nasa.earthdata/cmr-exchange-query "0.3.0-SNAPSHOT"]
                  [gov.nasa.earthdata/cmr-http-kit "0.2.0-SNAPSHOT"]
                  [gov.nasa.earthdata/cmr-jar-plugin "0.1.0"]
-                 [gov.nasa.earthdata/cmr-metadata-proxy "0.2.0-SNAPSHOT"]
+                 [gov.nasa.earthdata/cmr-metadata-proxy "0.2.2-SNAPSHOT"]
```
Comment (sxu123) [inline cmr-exchange/ous-plugin/project.clj]: Note: 0.2.1 was not used. some config params were removed from config.clj in 0.2.1.  This is why I modified the config.clj in a few places in this PR.

File: cmr-exchange/metadata-proxy/project.clj [hunk 2]
```diff
@@ -15,7 +15,7 @@
        ns
        "\u001B[35m]\u001B[33m λ\u001B[m=> "))
 
-(defproject gov.nasa.earthdata/cmr-metadata-proxy "0.2.1-SNAPSHOT"
+(defproject gov.nasa.earthdata/cmr-metadata-proxy "0.2.2-SNAPSHOT"
```
Comment (chris-durbin) [inline cmr-exchange/metadata-proxy/project.clj]: Have you published each of the updated library versions to clojars?

File: cmr-exchange/ses-plugin/src/cmr/sizing/formats.clj [hunk 3]
```diff
@@ -44,11 +44,18 @@
       (get-in [:umm :SizeEstimation :AverageSizeOfGranulesSampled])
       read-number))
 
+(defn- get-netcdf4-rate
+  "Gets the :Rate value for :NetCDF-4 format in avg-comp-info."
+  [avg-comp-info]
+  (some #(when (= "NetCDF-4" (:Format %)) (:Rate %)) avg-comp-info))
```
Comment (chris-durbin) [inline cmr-exchange/ses-plugin/src/cmr/sizing/formats.clj]: It's probably worth a `def` for the NetCDF-4 string.

Comment (sxu123) [reply cmr-exchange/ses-plugin/src/cmr/sizing/formats.clj]: Done.

File: cmr-exchange/metadata-proxy/project.clj (hunk 2 above)
Comment (sxu123) [reply cmr-exchange/metadata-proxy/project.clj]: Daniel will help me publishing them. Thanks @daniel-zamora

**Pull request merged**